src/Symfony/Component/Config/Builder/ConfigBuilderGenerator.php

```diff
@@ -426,9 +426,9 @@ private function getComment(BaseNode $node): string
             }
 
             if ($node instanceof EnumNode) {
-                $comment .= sprintf(' * @param ParamConfigurator|%s $value', implode('|', array_map(function ($a) {
+                $comment .= sprintf(' * @param ParamConfigurator|%s $value', implode('|', array_unique(array_map(function ($a) {
                     return var_export($a, true);
```

Comment:
in case of an enum, won't this miss a starting `\\` on PHP 8.1?

Reply:
Yes.

By testing it I realized that real enums are currently not supported because they can't be converted to string by array_unique so we are actually opening them here. So it's actually a bigger new feature and I need to look at it deeper.

Reply:
Enums are not supported by this node. I added a type check in `__construct()` to be sure we keep on disallowing values that won't pass `validateType()` at process time.